cdap-ui/app/directives/dag-plus/my-dag-ctrl.js

```diff
@@ -390,6 +393,7 @@ angular.module(PKG.name + '.commons')
 
     function addEndpointForConditionNode(endpointDOMId, endpointStyle, overlayLabel) {
```

Comment:
Are there other types of nodes with special endpoints? Are we able to handle adding error/alert nodes to Wrangler?

Reply:
Good point. I haven't added cypress markers to error & alert ports but it should be pretty easy if one wants to add them. I can add them if you feel it will belongs to this PR.